applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -1147,29 +1147,76 @@ set_app_screenshots(AppId, PathToScreenshotsFolder) ->
              ],
     update_screenshots(AppId, MA, SShots).
 
+%%------------------------------------------------------------------------------
+%% @doc Validate system schemas inside `system_schemas' database.
+%%
+%% @see kapps_maintenance:validate_system_configs/0
+%% @end
+%%------------------------------------------------------------------------------
 -spec check_system_configs() -> 'ok'.
 check_system_configs() ->
     _ = [lager:warning("System config ~s validation error:~p", [Config, Error])
          || {Config, Error} <- kapps_maintenance:validate_system_configs()
         ],
     'ok'.
 
+%%------------------------------------------------------------------------------
+%% @doc Updates system schemas using files in Crossbar `priv' folder during
+%% start up.
+%%
+%% This is called by {@link db_init/0} during Crossbar start up.
+%% @end
+%%------------------------------------------------------------------------------
 -spec update_schemas() -> 'ok'.
 update_schemas() ->
-    kz_datamgr:suppress_change_notice(),
+    kz_datamgr:enable_change_notice(),
```

Comment:
@icehess move ` kz_datamgr:suppress_change_notice` to `db_init_schemas` before calling `update_schemas`. `enable_change_notice` is by default and usually used after operations that fire `changes`. but we may look at the impact of removing it completely, i think `kz_datamgr:revise_docs_* `  does't update the doc if the content doesn't change.